target/linux/ath79/image/generic.mk

```diff
@@ -995,6 +1006,28 @@ define Device/devolo_magic-2-wifi
 endef
 TARGET_DEVICES += devolo_magic-2-wifi
 
+define Device/dlink_covr-p2500-a1
+  $(Device/loader-okli-uimage)
+  SOC := qca9563
+  DEVICE_VENDOR := D-Link
+  DEVICE_MODEL := COVR-P2500
+  DEVICE_VARIANT := A1
+  DEVICE_PACKAGES := kmod-ath10k-ct ath10k-firmware-qca9888-ct \
+	open-plc-utils open-plc-utils-hpavkey open-plc-utils-modpib \
+	open-plc-utils-plchost open-plc-utils-plctool
```

Comment:
The open-plc applications are not part of the main openwrt repository, but the feed, please remove them here.

Reply:
Fixed